dnf-plugins-extras.spec

```diff
@@ -358,7 +358,7 @@ Snapper Plugin for DNF, Python 3 version. Creates snapshot every transaction.
 %package -n python-dnf-plugins-extras-tracer
 Summary:        Tracer Plugin for DNF
 Requires:       python-dnf-plugins-extras-common = %{version}-%{release}
-Requires:       tracer
+Requires:       tracer > 0.6.11
```

Comment:
Please change this to require the Python module package (`python2-tracer`) rather than `tracer`.